<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>Fabian Hia CV</title>
    </head>






    <body>
        <h1>Fabian HIA</h1>
        <p>
            <b>Molecular & Cellular Biologist | Bioinformatician</b><br />
        </p>
        
        <img src="./images/Email Icon.png" height="12" alt="Email icon"/>
        <b>&nbsp&nbspfabianhia@gmail.com</b><br />
        
        <img src="./images/Globe Icon.png" height="15" alt="Globe icon"/>
        <b>&nbsp&nbsp&nbspSingaporean (currently based in Singapore)</b><br />
        
        <img src="./images/Mouse Icon.png" height="15" alt="Mouse icon"/>
        <b>Passed RCULA for mice (NUS)</b><br />
        <br />
        <a href="https://pubmed.ncbi.nlm.nih.gov/?term=fabian+hia">Full list of scientific publications</a>
        <br />
        
        <p>
            <b>Linkedin Profile</b><br />
            <a href="https://www.linkedin.com/in/fabianhia/"><img src="./images/linkedin_qr.JPG" height="100" alt="Linkedin QR code"/></a>
        </p>
        
        
        <hr />
        
        <!-- Executive Summary Section -->
            <h2>Executive Summary</h2>
            <ul>
                <li>Molecular and Cellular Biologist with 10 years of international and diverse wetlab as well as bioinformatics experience, driven by a passion for scientific exploration at the cellular and molecular level to contribute to breakthrough research</li>
                <br />
                <li>Proficient in molecular biology techniques, immunoblot assays, genomics, transcriptomics, mammalian and bacterial cell cultures, next-generation-sequencing, CRISPR-Cas9 techniques for performing impactful research on cellular processes and molecular interactions</li>
                <br />
                <li>In-depth knowledge of codon optimization and mRNA stability, leading to the design of highly efficient gene constructs for diverse applications and enriching gene regulation insights for therapeutic strategies</li>
                <br />
                <li>Venturing into R and Python programming as well as machine learning to enhance biological data analysis and visualization capabilities, adding valuable tools to the scientific toolkit</li>
                <br />
                <li>Delving into web development - HTML and CSS</li>
                <br>
                <li>Embraces challenges with a growth mindset, eager to explore cutting-edge methodologies and technologies for future scientific advancement</li>
                <br />
                <li>Thrives in collaborative environments, valuing teamwork and diverse perspectives for holistic problem-solving</li>
            </ul>
        
        
        <hr />
        
        <!-- Skills and Proficiencies -->
            <h2>Skills and Proficiencies</h2>
            <ul>
                <li>Molecular and Cellular techniques</li>
                <li>Mammalian and Bacterial assasys</li>
                <li>Cell-based assays</li>
                <li>Immunoblotting</li>
                <li>Codon optimization</li>
                <li>CRISPR-Cas9</li>
                <li>Bioanalytical chemistry (HPLC)</li>
                <li>Genomics and Transcriptomics</li>
                <li>Bioinformatics (NGS, Data mining)</li>
                <li>Programming for data science
                    <ul>
                    <li>Python</li>
                    <li>R</li>
                    <li>UNIX</li>
                    <li>HTML5</li>
                    </ul>
                </li>
            </ul>
            <hr />
        
        <!-- Work Experience -->
            <h2>Work Experience</h2>
                <h3>Singapore-MIT Alliance for Research and Technology</h3>
                <i><u>Senior Postdoctoral Associate</u></i><br />
                Nov 2023 - Present
                <ul>
                    <li>Development of a cell culture vessel-agnostic platform for re-engineering translation in cells to improve protein yields by leveraging on the phenomenon of stress-induced re-programming</li>
                    <li>Designed and executing POC experiments for mammalian codon optimized commmercailly-relevant proteins of interest</li>
                    <li></li>
                </ul>
        
                <i><u>Senior Postdoctoral Associate</u></i><br />
                Mar 2023 - Oct 2023
                <ul>
                    <li>Established a cell-line-based cancer drug discovery platform for massively parallel screening of chemical libraries against undrugged cancers</li>
                    <li>Developed a bioinformatics pipeline for evaluation of NGS results derived from massively parallel screens</li>
                </ul>
        
                <h3>Bioinformatics Consultant</h3>
                <i><u>Theonys Inc.</u></i><br />
                May 2022 - Dec 2022
                <ul>
                    <li>Triaging of target oncogenes and tumour suppressor genes by:</li>
                    <li>Transcriptome, hazard ratio and survival curve analysis of cancer patient databases from the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA)</li>
                    <li>Cancer cell line data mining and analysis from the DEPMAP portal (Cancer Dependency Map) for KO/KD essentiality, disease association, and gene ontology</li>
                    <li>Pathway and gene ontology analyses of codon-enriched oncogenes and proto-oncogenes</li>
                </ul>
        
                <h3>Kyoto University, Graduate School of Medicine, Dept. of Medicinal Chemistry, Japan</h3>
                <i><u>Japan Society for the Promotion of Science (JSPS), International Fellowship, Post-doctorate Research Fellow</u></i><br />
                Apr 2020 - Sep 2022
                <ul>
                    <li>Engineered reporters for a cell sorting-based CRISPR-Cas9 screening platform to identify
                    regulators of codon optimality</li>
                    <li>Verified a transcriptome-wide bias in codon optimality in human cell lines using next-
                        generation deep sequencing techniques and multivariate statistical analysis as well as
                        dimensional complexity reduction techniques which lead to a lead author publication in
                        <b>[F. Hia, EMBO Rep, 2019]</b></li>
                    <li>Ribosome- and disome profiling to establish the role of RBPs on ribosome stalling and
                        collision</li>
                    <li>Generated dominant-negative helicase-dead mutant helicases for reconstitution into
                        stable cell lines to investigate the RNA binding mechanisms of a DEAD-box helicase DHX29,
                        in vitro</li>
                    <li>Integration of bioinformatics, biostatistics and wet-lab data for hypothesis generation and
                        target identification of RBPs in various KO cells (GSEA, gene ontology analysis among
                        others)</li>
                    <li>Interdisciplinary project collaborations:
                        <ul>
                            <li>Bioinformatics analyses of RNAseq, RNA-immunoprecipitation of regulators of
                                immune response to elucidate novel pathways for therapeutic intervention of excess
                                inflammation. Co-authored publications: <b>[M. Yoshinaga, Nat Commun, 2022],
                                [YK Chong, Sci Signal, 2022], [K. Akaki, Elife, 2021], [T. Mino, Nucleic Acids Res,
                                2019], [X. Cui, J Immunol, 2017]</b></li>
                            <li>Inference of ligand-receptor activities from modelling of transcriptomics data derived
                                from alveolar macrophages and pulmonary arteries to identify factors which
                                engender pulmonary arterial hypertension (via NicheNet). Co-authored publications:
                                <b>[A. Yaku, Circulation, 2022]</b></li>
                            <li>Motif analysis of novel RNA-binding proteins via CIMS analysis to identify protein-
                                RNA crosslink sites and protein-RNA interactions at single-nucleotide resolution</li>
                            <li>Bioinformatics analysis to determine transcriptome-wide RNA structure dynamics in
                                mammalian cell lines as part of a group effort to identify targetable stem-loops for
                                morpholino-oligo based therapeutic intervention in controlling excess inflammation.
                                Co-authored publications: <b>[KM. Tse, Sci Transl Med, 2022]</b></li>
                        </ul>
                    </li>
                </ul>
        
                <h3>Singapore-MIT Alliance for Research and Technology</h3>
                <i><u>Laboratory Technologist (RNA Modifications Group)</u></i><br />
                Jun 2014 - Mar 2015<br />
                Jul 2011 - Aug 2012
                <ul>
                    <li>Isolated and purifying individual RNA species using High Performance Liquid
                        Chromatography (HPLC) and size exclusion columns which lead to several co-authored
                        publications: <b>[YH. Chionh, Nucleic Acids Res, 2013], [YH. Chionh, Nat Commun, 2016]</b></li>
                        <li>Developed and optimized a solid-phase RNA extraction method recover the entire
                        spectrum of non-coding RNA from <i>Mycobacterium bovis</i> BCG which lead to a lead author
                        publication <b>[F. Hia, Nucleic Acids Res, 2015]</b> and a co-author publication <b>[WM. Cai,
                        Methods Enzymol, 2015]</b></li>
                        <li>Executed Liquid Chromatography-Mass Spectrometry experiments which lead to the
                        publication of a co-authorship publication <b>[H. Dong, PLoS Pathog, 2012]</b></li>
                        <li>Assisted in optimizing a method for the purification of individual non-coding RNA species
                        from <i>Mycobacterium bovis</i> BCG using HPLC 2-dimensional size-exclusion techniques</li>
                        <li>Documented procedures and prepared orders and quotations for the group
                    <li>Prepared and maintained inventory for the Mass spectrometry facility</li>
                </ul>
        
        
        <!-- Education -->
            <hr />
        
            <h2>Education</h2>
                <h3>Kyoto University, Graduate School of Medicine, Dept. of Medicinal Chemistry, Japan</h3>
                <i><u>Doctor of Philosophy in Medicine, Monbukagakusho (MEXT) Scholar</u></i><br />
                Apr 2016 - Mar 2020
                <ul>
                    <li>Investigated the role of RNA binding proteins in the regulation of mRNA via codon
                        optimality with regard to the human immune response and evolution of cancers</li>
                    <li>Designed a tractable codon optimization/deoptimization system to generate codon-
                        biased reporters of varying optimalities which resulted in a lead author publication in
                        <b>[F. Hia, EMBO Rep, 2019]</b></li>
                    <li>Verified reporter translation efficiencies via a combination of cell sorting, ELISA,
                        traditional immunoblot and PCR-based techniques</li>
                    <li>Identified ILF2-ILF3 heterodimers as AU-rich binding factors through integration of
                        bioinformatics analyses of online databases, proteomics-based methods, gene
                        knockdown (siRNA) and overexpression systems</li>
                    <li>RNA sequence motif analysis to ascertain binding sites of ILF2-ILF3 heterodimers</li>
                    <li>Demonstrated that ILF2-ILF3 heterodimers act in concert to degrade AU-rich transcripts
                        in a translation-independent fashion via translational- and transcriptional inhibitors</li>
                </ul>
        
                <h3>Kyoto University, Institute for Virus Research, Laboratory of Infection and Prevention, Japan</h3>
                <i><u>Monbukagakusho (MEXT) Scholarship Research Student</u></i><br />
                Apr 2015 - Mar 2016
                <ul>
                    <li>Demonstrated the presence of a transcriptome-wide system of codon bias in humans
                        via application of multi-variate statistical techniques involving dimensional and
                        complexity reduction techniques, hierarchical clustering and bioinformatic analyses
                    <li>Characterized synonymous substitutions in mRNAs of tumour suppressor genes and
                        (proto)oncogenes to assess feasibility of utilizing (de)stabilizing substitutions as possible
                        prognostic biomarkers of cancer progression</li>
                </ul>
                
                <h3>University of Basel, Swiss Tropical and Public Health Institute, Switzerland</h3>
                <i><u>Master of Science in Infection Biology</u></i><br />
                Sep 2012 - Feb 2014
                <ul>
                    <li>Established, characterized, and optimized the infection of THP-1 cells by
                        <i>Mycobacterium bovis</i> BCG as part of a lipidomics study on mycobacterial infection</li>
                    <li>Characterized the immune responses of the human THP-1 macrophage-like cell line to
                        Lipopolysaccharide and Kdo2 Lipid A</li>
                    <li>Investigated growth dynamics of <i>Mycobacterium bovis</i> BCG in non-hydrolyzable
                        detergent Tyloxapol as an alternative to Tween 80</li>
                </ul>
        
                <h3>National University of Singapore (NUS), Singapore</h3>
                <i><u>Bachelor of Science (Honours) Life Sciences with concentrations in Biomedical Science</u></i><br />
                Aug 2007 - Jun 2011
                <h3>National University of Singapore (NUS), Department of Microbiology, Singapore</h3>
                <i><u>Bachelor of Science Honours Project</u></i><br />
                Jun 2010 - May 2011
                <ul>
                    <li>Assessed the effects of administration of Fibroblast Growth Factor 10 (FGF10) in the
                        lower respiratory tract of mice following infection by Influenza A/PR/8 (H1N1) with the
                        aim of therapeutically promoting lung repair and regeneration</li>
                    <li>Verified the lung repair and regeneration of mice via immunohistological analyses
                        of PFFE lung histology slides following influenza challenge via intra-tracheal instillation</li>
                </ul>
            <hr />
        
        <!-- Fellowships, Scholarships and Awards -->
        
            <h2>Fellowships, Scholarships and Awards</h2>
                <ul>
                    <li><b>Japan Society for the Promotion of Science (JSPS), International Fellowship, Apr 2020 - Mar 2022</b></li>
                    <li><b>RNA 2020 On-line, RNA Society Poster Award, Jul 2020</b></li>
                    <li><b>2015 Monbukagakusho (MEXT) Japanese Government Scholarship, Apr 2015 - Mar 2020</b></li>
                    <li><b>The 16th International Student Seminar, Outstanding Oral Presentation (Short Talk), Mar 2018</b></li>
                    <li><b>RNA Frontier Meeting 2018, MBL Best Presentation Award, Sep 2018</b></li>
                </ul>
        
            <hr />
        
        <!-- Language Proficiency -->
        
            <h2>Language Proficiency</h2>
                <ul>
                    <li><b>English - First (native) language. Proficient in conversational, writing and reading</b></li>
                    <li><b>Mandarin Chinese - Intermediate in conversational, in writing and reading</b></li>
                    <li><b>Japanese - Intermediate in conversational, writing and reading (Passed JLPTN3)</b></li>
                </ul>
        
                <hr />
        
        
        
        





        <footer>
            <p>© 2024 Fabian Hia. All rights reserved.</p>
        </footer>

    </body>
</html>



